cypress/integration/plugins/security-dashboards-plugin/missing_role_button_logout.js

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+if(Cypress.env('SECURITY_ENABLED')){
```

Comment:
```suggestion
if (Cypress.env('SECURITY_ENABLED')) {
```
I wonder if you have prettier editor plugin? It helps to auto-format the code if you want.